.cursor/rules/autofill-add-site-specific-settings.mdc

```diff
@@ -0,0 +1,9 @@
+---
+alwaysApply: false
```

Comment:
Add a description and it should then be picked up when folk type the relevant stuff.